src/sentry/static/sentry/app/views/events/utils/eventsRequest.tsx

```diff
@@ -112,45 +157,38 @@ class EventsRequest extends React.PureComponent {
     end: null,
     interval: '1d',
     limit: 15,
-    getCategory: i => i,
     query: '',
 
     includePrevious: true,
     includeTransformedData: true,
   };
 
-  constructor(props) {
-    super(props);
-    this.state = {
-      reloading: false || props.loading,
-      timeseriesData: null,
-    };
-  }
+  state = {
+    reloading: false || !!this.props.loading,
```

Comment:
Can we go ahead and drop the `false ||`?

Reply:
oops